Makefile.am

```diff
@@ -96,7 +96,9 @@ pkginclude_HEADERS = \
 	src/libcollectdclient/collectd/server.h \
 	src/libcollectdclient/collectd/types.h
 
-lib_LTLIBRARIES = libcollectdclient.la
+lib_LTLIBRARIES = \
+	libcollectd.la \
```

Comment:
So, this adds a new library installed into /usr/lib/ (or whatever it is on the target system) which means we'll have to start doing proper versioning, etc. I'd like @octo to comment on that as well.